### PR TITLE
Cockpit 306

### PR DESCRIPTION
**Pull request by martinpitt**
Preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-306.html

 - [x] add bodhi URLs for c-ostree (release still running)

### CODE REVIEW

**Comment by martinpitt** [comment]
I cropped and optipng'ed the two ostree dialogs

File: _posts/2023-11-29-cockpit-306.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Kdump: Add Ansible/shell automation
+
+The Kdump page can generate an [Ansible](https://www.ansible.com/automation) role or a shell script for replicating the current kdump configuration to other machines.
+
+![screenshot of add ansible/shell automation]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation.png)
+
+![screenshot of add ansible/shell automation]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation-2.png)
+
+## OSTree: Redesign with new features: status, clean up, reset, and pin
```
Comment (martinpitt) [inline _posts/2023-11-29-cockpit-306.md:25]: This is too long, it's a bit ugly when the title wraps to two lines. @garrett: Perhaps just "OSTree: Redesign with new features" and we leave the details to the paragraphs?

Comment (garrett) [reply _posts/2023-11-29-cockpit-306.md:25]: Sure!

```suggestion
## OSTree: Redesign, with new features
```

**Comment by garrett** [comment]
> I cropped and optipng'ed the two ostree dialogs

FWIW: `optipng` has been deprecated for years; `oxipng` is a Rust fork that is the "spiritual successor" that is maintained and is a lot better in every way.

However, optipng was already quite good; we don't need to shave off a few additional (theoretical) bytes or even kilobytes. Just pointing it out for the future. :wink:

File: _posts/2023-11-29-cockpit-306.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+We release regularly.
+
+Here are the release notes from Cockpit 306, cockpit-machines 303, cockpit-podman 82, and cockpit-ostree 198.1:
+
+
+## Kdump: Add Ansible/shell automation
+
+The Kdump page can generate an [Ansible](https://www.ansible.com/automation) role or a shell script for replicating the current kdump configuration to other machines.
+
+![screenshot of add ansible/shell automation]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation.png)
```
Comment (garrett) [inline _posts/2023-11-29-cockpit-306.md:21]:
```suggestion
![screenshot of the kdump page with an Ansible button]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation.png)
```

File: _posts/2023-11-29-cockpit-306.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+![screenshot of add ansible/shell automation]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation-2.png)
```
Comment (garrett) [inline _posts/2023-11-29-cockpit-306.md:23]:
```suggestion
![screenshot of shell automation dialog with the "Ansible" tab selected and the "shell script" tab unselected]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation-2.png)
```

File: _posts/2023-11-29-cockpit-306.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+Deployments can be pinned to persist even when new deployments trigger an auto-cleanup and unpinned.
+
+![screenshot showing a pinned deployment]({{ site.baseurl }}/images/306-ostree-redesign-with-new-features-status-clean-up-reset-and-pin-4.png)
+## Machines: Change "Add disk" default behavior
+
+The "Always attach" Persistent option will now be set by default.
+
+## Podman: Delete intermediate images
+
+Intermediate images have no tags or other identifiers, they are usually intermediate layers from building container images, or leftovers from moved tags. These can now be deleted:
```
Comment (garrett) [inline _posts/2023-11-29-cockpit-306.md:49]:
```suggestion
Intermediate images have no tags or other identifiers. They are usually intermediate layers from building container images or leftovers from moved tags. Intermediate images can now be deleted within Cockpit Podman.
```

File: _posts/2023-11-29-cockpit-306.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+![screenshot of delete intermediate images]({{ site.baseurl }}/images/306-podman-delete-intermediate-images.png)
```
Comment (garrett) [inline _posts/2023-11-29-cockpit-306.md:51]:
```suggestion
![screenshot of the "delete intermediate images" modal dialog]({{ site.baseurl }}/images/306-podman-delete-intermediate-images.png)
```

File: _posts/2023-11-29-cockpit-306.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+![screenshot of add ansible/shell automation]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation.png)
+
+![screenshot of add ansible/shell automation]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation-2.png)
+
+## OSTree: Redesign with new features: status, clean up, reset, and pin
+
+The OSTree page for software updates has been redesigned and includes several new features.
+
+Update status is now listed in the "Status" card, and details about the current OSTree repository and branch are visible in the "OSTree source" card:
```
Comment (garrett) [inline _posts/2023-11-29-cockpit-306.md:29]:
```suggestion
Update status is now listed in the "Status" card, and details about the current OSTree repository and branch are visible in the "OSTree source" card.
```

File: _posts/2023-11-29-cockpit-306.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+![screenshot of status, clean up, reset, and pin]({{ site.baseurl }}/images/306-ostree-redesign-with-new-features-status-clean-up-reset-and-pin.png)
+
+Unused deployments and package cache can be removed in using the "clean up" action.
+![screenshot of status, clean up, reset, and pin]({{ site.baseurl }}/images/306-ostree-redesign-with-new-features-status-clean-up-reset-and-pin-2.png)
```
Comment (garrett) [inline _posts/2023-11-29-cockpit-306.md:34]:
```suggestion

![screenshot of the clean up modal dialog, with cache selected and deployments unselected by default]({{ site.baseurl }}/images/306-ostree-redesign-with-new-features-status-clean-up-reset-and-pin-2.png)
```

Alt text and blank separator line between text and image

File: _posts/2023-11-29-cockpit-306.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+![screenshot of add ansible/shell automation]({{ site.baseurl }}/images/306-kdump-add-ansible-shell-automation-2.png)
+
+## OSTree: Redesign with new features: status, clean up, reset, and pin
+
+The OSTree page for software updates has been redesigned and includes several new features.
+
+Update status is now listed in the "Status" card, and details about the current OSTree repository and branch are visible in the "OSTree source" card:
+
+![screenshot of status, clean up, reset, and pin]({{ site.baseurl }}/images/306-ostree-redesign-with-new-features-status-clean-up-reset-and-pin.png)
```
Comment (garrett) [inline _posts/2023-11-29-cockpit-306.md:31]:
```suggestion
![new design of the OSTree page, showing off the status card]({{ site.baseurl }}/images/306-ostree-redesign-with-new-features-status-clean-up-reset-and-pin.png)
```

**Review by garrett — CHANGES_REQUESTED** [review]
Several suggestions

**Comment by garrett** [comment]
We also need to crop that last dialog screenshot

**Pull request merged**